From 8aa755cc69a4050ee9aa630b71cc518297ba9b2e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20La=C3=9F?= <lass@mail.uni-paderborn.de>
Date: Tue, 4 Jun 2013 21:46:15 +0200
Subject: [PATCH 1/2] Adjust RedHat config and service files

Basically we reuse RedHat's config but we have to adjust some paths in it.
Also we don't need all of the stuff that's contained in RedHat's service files.

Change-Id: I4f35cd101b5cff24de8c86f8316fd7909f006760
---
 src/packaging/RedHat/openafs-cacheinfo      |  2 +-
 src/packaging/RedHat/openafs-client.service | 14 ++++++--------
 src/packaging/RedHat/openafs-server.service |  4 ++--
 3 files changed, 9 insertions(+), 11 deletions(-)

diff --git a/src/packaging/RedHat/openafs-cacheinfo b/src/packaging/RedHat/openafs-cacheinfo
index dd0fd05..1663c79 100755
--- a/src/packaging/RedHat/openafs-cacheinfo
+++ b/src/packaging/RedHat/openafs-cacheinfo
@@ -1 +1 @@
-/afs:/usr/vice/cache:100000
+/afs:/var/cache/openafs:100000
diff --git a/src/packaging/RedHat/openafs-client.service b/src/packaging/RedHat/openafs-client.service
index 936762e..ac479e7 100644
--- a/src/packaging/RedHat/openafs-client.service
+++ b/src/packaging/RedHat/openafs-client.service
@@ -4,14 +4,12 @@ After=syslog.target network.target
 
 [Service]
 Type=forking
-EnvironmentFile=/etc/sysconfig/openafs
-ExecStartPre=/bin/sed -n 'w/usr/vice/etc/CellServDB' /usr/vice/etc/CellServDB.local /usr/vice/etc/CellServDB.dist
-ExecStartPre=/bin/chmod 0644 /usr/vice/etc/CellServDB
-ExecStartPre=/sbin/modprobe openafs
-ExecStart=/usr/vice/etc/afsd $AFSD_ARGS
-ExecStop=/bin/umount /afs
-ExecStop=/usr/vice/etc/afsd -shutdown
-ExecStop=/sbin/rmmod openafs
+EnvironmentFile=/etc/conf.d/openafs
+ExecStartPre=/usr/bin/modprobe openafs
+ExecStart=/usr/bin/afsd $AFSD_ARGS
+ExecStop=/usr/bin/umount /afs
+ExecStop=/usr/bin/afsd -shutdown
+ExecStop=/usr/bin/rmmod openafs
 KillMode=none
 
 [Install]
diff --git a/src/packaging/RedHat/openafs-server.service b/src/packaging/RedHat/openafs-server.service
index 2d34bb0..400b900 100644
--- a/src/packaging/RedHat/openafs-server.service
+++ b/src/packaging/RedHat/openafs-server.service
@@ -3,8 +3,8 @@ Description=OpenAFS Server Service
 After=syslog.target network.target
 
 [Service]
-EnvironmentFile=-/etc/sysconfig/openafs
-ExecStart=/usr/afs/bin/bosserver $BOSSERVER_ARGS
+EnvironmentFile=-/etc/conf.d/openafs
+ExecStart=/usr/bin/bosserver $BOSSERVER_ARGS
 ExecStop=/usr/bin/bos shutdown localhost -wait -localauth
 
 [Install]
-- 
1.8.3.3

